src/types/configuration/user-extractor-config.ts

```diff
@@ -0,0 +1,8 @@
+import { Static, Type } from "@sinclair/typebox";
+
+export const userExtractorConfigurationType = Type.Object({
+  enabled: Type.Boolean({ default: true }),
+  redeemTask: Type.Boolean({ default: true }),
```

Comment:
What is redeem task?

Reply:
If you disable this, users won't be able to collect the reward associated to the issue